ts/features/bonusVacanze/components/BonusCardComponent.tsx

```diff
@@ -155,7 +155,13 @@ const BonusCardComponent: React.FunctionComponent<Props> = (props: Props) => {
 
               <MenuOptions>
                 <MenuOption
-                  onSelect={() => clipboardSetStringWithFeedback(bonus.id)}
+                  onSelect={() =>
+                    clipboardSetStringWithFeedback(
+                      addEvery(bonus.id, " ", 4)
+                        .trim()
+                        .toString()
```

Comment:
```suggestion
                        
```
Since it is already a string.
Btw what do you think if we create a function in bonus/utils, i.e `getBonusCode(bonus: Bonus) : string` ?
In this way we centered how we format the string code

Reply:
Do you mean a function to centralize the formatting of the bonus code?

Yes it could be a good point since it is shared at least by 3 components at the moment.

Reply:
@CrisTofani Yes we could put it in `bonusVacanze/utils/bonus.ts`
something like this: `getBonusCodeFormatted(bonus: BonusActivationWithQrCode) : string`

Reply:
Added in 0e0a51a